benefits/templates/admin/base.html

```diff
@@ -0,0 +1,56 @@
+{% extends "admin/base.html" %}
+{% load i18n static %}
+
+{% block dark-mode-vars %}
+{% endblock dark-mode-vars %}
+
+{% block extrahead %}
+  {% include "admin/includes/favicon.html" %}
+  {% include "core/includes/bootstrap-css.html" %}
+  <link href="{% static "css/admin/overrides.css" %}" rel="stylesheet">
```

Comment:
This is purely a code style question for everyone @angela-tran @lalver1 @thekaveman:

Is there an advantage to making a template for `favicon.html`, for example, instead of just directly writing `<link href="{% static "img/favicon.ico" %}" rel="icon" type="image/x-icon" />` here? I found this hard to read.

So here, I just wrote `<link href="{% static "css/admin/overrides.css" %}" rel="stylesheet">` here, instead of making an HTML template just to put the CSS file in.

As someone reading this file for the first time ever, I found it a hard to read b/c I had to reference many files just to see what was ultimately rendered.

Reply:
Yeah I think I tend to agree, this doesn't feel worth the extra include. 

If we already had an include that was using the app, then reusing it for admin could make sense. But in the app we just have the `<link>` directly in the template.